### PR TITLE
display flash message instead of highlighting element

### PR DESCRIPTION
**Pull request by vkghub**
ME-180058554

### CODE REVIEW

File: features/financial_assistance/step_definitions/applicant_personal_info_page_steps.rb [hunk 1]
```diff
@@ -37,6 +37,11 @@
   find(:xpath, "//div[@class='selectric-scroll']")
 end
 
+Then(/form should not submit due to required relationship error$/) do
+  expect(page).to have_content 'Select relationship type'
+end
+
 And(/user fills in the missing relationship$/) do
+  find("span", :text => "choose").click
```
Comment (markgoho) [inline features/financial_assistance/step_definitions/applicant_personal_info_page_steps.rb:45]: 1. please use class selector or page object
2. don't hard code english into specs

**Review by markgoho — CHANGES_REQUESTED** [review]

File: features/financial_assistance/step_definitions/applicant_personal_info_page_steps.rb (hunk 1 above)
Comment (vkghub) [reply features/financial_assistance/step_definitions/applicant_personal_info_page_steps.rb:45]: thank you mark, I was already working on it. Changes will be available in next commit.

**Review by bill-transue — APPROVED** [review]
Change highlighting element to displaying a flash message. Thanks @vkghub!

**Review by markgoho — APPROVED** [review]

**Review by markgoho — CHANGES_REQUESTED** [review]
please provide working ticket reference

**Review by markgoho — APPROVED** [review]

**Pull request merged**